app/src/main/java/com/bucketlist/dto/BucketListItem.kt

```diff
@@ -3,8 +3,14 @@ package com.bucketlist.dto
 import androidx.room.Entity
 import androidx.room.PrimaryKey
 
-@Entity(tableName="bucketlistitem")
-data class BucketListItem(var name: String = "", @PrimaryKey var itemID: Int = 0,) {
+@Entity(tableName="bucketListItem")
+
+/**
+* @param name: What the bucket list item is called
+* @param itemID: Unique Identifier of this bucket list item.
+* */
```

Comment:
KDoc is a good idea.  Add a bit more to this to describe what the user can't tell just by reading the code.